extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -190,6 +190,15 @@ export class AmpAdUIHandler {
   }
 
   /**
+   * Public method for forcing the collapse of a slot.
+   */
+  forceNoContentUI() {
```

Comment:
instead of introducing a new public method, can we make change to the `setDisplayState(state)` method?